src/protean/core/entity.py

```diff
@@ -214,9 +232,14 @@ def all(self):
         # order_by clause must be list of keys
         order_by = model_cls.opts_.order_by if not self._order_by else self._order_by
 
+        # Breakdown filters
+        self._restructured_filters = self._restructure_filters(adapter, self._filters)
```

Comment:
We don't really need to save this in the object, if we ever need it we can just call the function to get it back. We can then just call the attrs as `filters` and `excludes`